.travis.yml

```diff
@@ -1,10 +1,24 @@
 language: node_js
 node_js:
   - "node"
+addons:
+  chrome: stable
+before_install:
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+install:
+  - npm install
+script:
+  - npm run build
 deploy:
```

Comment:
@anumehaagrawal & @BaaniLeen - this deploy section in the travis.yml file is what takes care of the deployment to heroku.